specification/ai/Azure.AI.Projects/evaluations/models.tsp

```diff
@@ -12,6 +12,70 @@ using TypeSpec.Versioning;
 
 namespace Azure.AI.Projects;
 
+@doc("OpenAI definition: Eval output results.")
+@added(Versions.v2025_10_15_preview)
+model EvalRunOutputItem {
+  /** The type of the object. Always "eval.run.output_item". */
+  object: "eval.run.output_item" = "eval.run.output_item";
+
+  /** Unique identifier for the evaluation run output item. */
+  id: string;
+
+  /** The identifier of the evaluation run associated with this output item. */
+  run_id: string;
+
+  /** The identifier of the evaluation group. */
+  eval_id: string;
+
+  /** Unix timestamp (in seconds) when the evaluation run was created. */
+  created_at: integer;
+
+  /** The status of the evaluation run. */
+  status: string;
+
+  /** The identifier for the data source item. */
+  datasource_item_id: integer;
+
+  /** Details of the input data source item. */
+  datasource_item: {};
+
+  /** A list of results from the evaluation run. */
+  results: {}[];
+
+  /** A sample containing the input and output of the evaluation run. */
+  sample: {};
+}
+
+@doc("OpenAI definintion: Eval inline file content or jsonl input data.")
+@added(Versions.v2025_10_15_preview)
+model EvalJsonlFileContentSource {
+  /** The type of jsonl source. Always `file_content`. */
+  type: "file_content" = "file_content";
+
+  /** The content of the jsonl file. */
+  content: {
+    item: {};
+    sample?: {};
+  }[];
+}
+
+@doc("Eval jsonl input data.")
+@added(Versions.v2025_10_15_preview)
+model CreateEvalJsonlRunDataSource {
+  /** The type of data source. Always `jsonl`. */
+  type: "jsonl" = "jsonl";
+
+  /** Determines what populates the `item` namespace in the data source. */
+  source: EvalJsonlFileContentSource;
+}
+
+@doc("OpenAI Input data source definition for eval.")
```

Comment:
Stick to doc comments `/** */` instead of `@doc`

Reply:
the problem is this repo is already @doc and its better to just stick to one convention especially if it is already pre-established right?

Reply:
any new spec migrated to Typespec use doc comments(e.g. latest migration [pr](https://github.com/Azure/azure-rest-api-specs/pull/37847/)), what other specs have done in the past is not a reason to not follow the best practices today.